params.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strconv"
 )
 
-// Params represents a set of parameters that gets passed to a request.
+// Params represents a set of parameters that gets past to a request.
```

Comment:
I think this was correct as-is (passed, not past).

Reply:
agree, will revert